src/assets/style.css

```diff
@@ -50,6 +50,7 @@
   right: 0;
   left: 0;
   pointer-events: none;
+  margin-bottom: 5px;
```

Comment:
this would affect subtitles for every engine, right ? does it look ok visually ?

Reply:
this file affects only the dash text displayer